ui/components/app/selected-account/selected-account.component.js

```diff
@@ -11,8 +11,16 @@ import { getEnvironmentType } from '../../../../app/scripts/lib/util';
 import { ENVIRONMENT_TYPE_POPUP } from '../../../../shared/constants/app';
 import CustodyLabels from '../../institutional/custody-labels/custody-labels';
 ///: END:ONLY_INCLUDE_IF
-import { Icon, IconName, IconSize } from '../../component-library';
-import { IconColor } from '../../../helpers/constants/design-system';
+import { Icon, IconName, IconSize, Text } from '../../component-library';
+import {
+  IconColor,
+  TextVariant,
+  TextColor,
+  TextAlign,
+  Display,
+  AlignItems,
+  BlockSize,
```

Comment:
import `FontWeight` here

```suggestion
  BlockSize,
  FontWeight,
```